doc/README.build_image.md

```diff
@@ -0,0 +1,50 @@
+
+# README for build_image.py
+
+This `build_image.py` script is used to create a runtime docker image with Open-CE
+packages (generated from `build_env.py`) installed in it. This script takes two arguments
+as an input - local conda channel and conda environment file which are the output of `build_env.py`
+script.
+
+For example,
+```shell
+    open-ce/open-ce/build_image.py --local_conda_channel=./condabuild
+           --conda_env_file=open-ce-conda-env-py3.7-cuda-openmpi.yaml
+```
+
+`local_conda_channel` is the output folder that has all of the conda packages built within it. It has to
+be present in the directory from where this `build_image.py` script is run.
+`conda_env_file` is the conda environment file generated from build_env.py. 
+
+A docker image created has a conda environment that has all the packages mentioned in the 
+conda environment file, installed in it. The local conda channel being passed is also copied into the
+image to enable users to create their custom environments.
+
+Note that the image will not necessarily have all the Open-CE packages installed.
+The packages to be installed strictly depends on the conda environment file which is used to build the image.
+For more information on how conda environment files are generated and their content, please see 
+[`doc/README.build_env.md`](doc/README.build_env.md).
+
+So, the ideal sequence of getting Open-CE packages built and installed in a container should be
+1. `build_env.py` 
```

Comment:
Shall we add one liner description too here for #1 and #2.

Something like:
1. Build packages and target conda environment file using `build_env.py`.
2. Create Docker image using `build_image.py`

Reply:
Yes, sounds good.